File: Ad-Hoc/URI 1893.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        int inicio = int.Parse(input[0]);
        int final = int.Parse(input[1]);
        if (inicio >= 0 && final <= 2) {
            Console.WriteLine("nova");
        } else if (final > inicio && final <= 96) {
            Console.WriteLine("crescente");
        } else if (inicio >= final && final <= 96) {
            Console.WriteLine("minguante");
        } else { 
            Console.WriteLine("cheia");
        }
    }
    
}

File: Ad-Hoc/URI 1893.dart
import 'dart:io';

void main() { 
  List<String> input = stdin.readLineSync().split(" ");
  int inicio = int.parse(input[0]);
  int finau = int.parse(input[1]);
  if (inicio >= 0 && finau <= 2) {
      print("nova");
  } else if (finau > inicio && finau <= 96) {
      print("crescente");
  } else if (inicio >= finau && finau <= 96) {
      print("minguante");
  } else { 
      print("cheia");
  }
}